pynmea2/types/proprietary/grm.py

```diff
@@ -1,27 +1,32 @@
 # Garmin
 
+from decimal import Decimal
+
 from ... import nmea
 
 class GRM(nmea.ProprietarySentence):
     sentence_types = {}
+
     def __new__(_cls, manufacturer, data):
         name = manufacturer + data[0]
         cls = _cls.sentence_types.get(name, _cls)
         return super(GRM, cls).__new__(cls)
 
     def __init__(self, manufacturer, data):
-        self.sentence_type = manufacturer + data[1]
-        super(GRM, self).__init__(manufacturer, data[2:])
+        self.sentence_type = manufacturer + data[0]
+        super(GRM, self).__init__(manufacturer, data)
+
 
 class GRME(GRM):
     """ GARMIN Estimated position error
     """
     fields = (
-        ("Estimated Horiz. Position Error", "hpe"),
+        ("Subtype", "subtype"),
+        ("Estimated Horiz. Position Error", "hpe", Decimal),
         ("Estimated Horiz. Position Error Unit (M)", "hpe_unit"),
-        ("Estimated Vert. Position Error", "vpe"),
+        ("Estimated Vert. Position Error", "vpe", Decimal),

```

Comment:
No problem, and thanks for helping out, I don't have these proprietary GPS devices, and pull requests are easy :)